src/components/NavBar/MenuDropdown.tsx

```diff
@@ -127,7 +127,7 @@ export const MenuDropdown = () => {
   return (
     <>
       <Box position="relative" ref={ref}>
-        <NavIcon isActive={isOpen} onClick={toggleOpen}>
+        <NavIcon isActive={isOpen} onClick={toggleOpen} label={`${isOpen ? 'hide' : 'show'} resources`}>
```

Comment:
These should be sentence-cased and translated:

    import { t, Trans } from '@lingui/macro'
    ...
    <NavIcon isActive={isOpen} onClick={toggleOpen} label={isOpen ? t`Hide resource` : t`Show resources`}>